includes/languages/english/lang.accessibility.php

```diff
@@ -0,0 +1,8 @@
+<?php
+$define = [
+    'NAVBAR_TITLE' => 'Accessibility Statement',
+    'HEADING_TITLE' => 'Accessibility Statement',
+    'TEXT_INFORMATION' => 'Your Accessibility Statement should be on this page.',
```

Comment:
This `TEXT_INFORMATION` line is never used. 
Probably ideal to remove it, in favor of the storeowner updating the actual page content via the Define Pages Editor.